src/compress.ts

```diff
@@ -72,5 +72,24 @@ function isTextMime(mimeType: string) {
 }
 
 function isCompressableMime(mimeType: string) {
-  return /image|text|font|json|xml|javascript/.test(mimeType);
+  const compressibleMimeTypes = [
+    'atom',
+    'css',
+    'eot',
+    'htc',
+    'html',
+    'ico',
+    'js',
+    'json',
+    'mjs',
+    'otf',
+    'rss',
+    'svg',
+    'text',
+    'ttf',
+    'webmanifest',
+    'xml',
+  ];
+
```

Comment:
I think we can extract this from the function so we don't redeclare the array each time:

```suggestion
```